src/main/java/io/github/thebusybiscuit/sensibletoolbox/utils/MaterialConverter.java

```diff
@@ -0,0 +1,58 @@
+package io.github.thebusybiscuit.sensibletoolbox.utils;
+
+import java.util.Optional;
+
+import org.bukkit.Material;
+
+import lombok.NonNull;
+
+public final class MaterialConverter {

```

Comment:
I feel like this class should not only be documented but also deprecated, seems like a quick and dirty fix but needs to be improved in the long run

Reply:
Because of the new fancy material names we need something like this.
There is already "STBUtil" class with plenty methods like this, i could join them if you want me to.